internal/compass/compass.go

```diff
@@ -18,30 +18,19 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"os/exec"
 	"time"
 )
 
 const waitForRunningStateDuration = 10 * time.Second
 
 var errCompassExited = errors.New("MongoDB Compass process has exited")
 
-func binPath() string {
-	if p, err := exec.LookPath(compassBin); err == nil {
-		return p
-	}
-	return ""
-}
-
 func Detect() bool {
 	return binPath() != ""
 }
 
 func Run(username, password, mongoURI string) error {
 	path := binPath()
-	if path != compassBin {
-		path += compassBin
-	}
 
 	cmd := compassCmd(path, username, password, mongoURI)
```

Comment:
[nit] can we add an unit test on what happens when path == ""? hoping we can capture as we have windows unit tests running

Reply:
It will fail on line 36, because `cmd.Start` will fail.

We can add a unit test but we would have to introduce mocks and replace a lot of code. I rather do this in a separate PR.